test_copilot-chat.el

```diff
@@ -26,5 +28,10 @@
   (let ((copilot-chat-prompt-suffix "extra details"))
     (should (equal (test-major-mode 'python-mode)
                    "current programming language is: python, extra details"))
+    (should (equal (test-major-mode 'python-ts-mode)
+                   "current programming language is: python, extra details"))
     (should (equal (test-major-mode 'c-mode)
-                   "current programming language is: c, extra details"))))
+                   "current programming language is: c, extra details"))
+    (should (equal (test-major-mode 'c++-ts-mode)
```

Comment:
adding test from https://github.com/chep/copilot-chat.el/issues/50 and make sure test passed.